src/service/referral-api/acceptance-tests/FamilyHubs.Referral.Api.AcceptanceTests/Configuration/ConfigAccessor.cs

```diff
@@ -8,10 +9,14 @@ public class ConfigAccessor
 
     private static IConfigurationRoot GetIConfigurationRoot()
     {
-        return _root ??= new ConfigurationBuilder()
+        var builder = new ConfigurationBuilder()
             .AddJsonFile("appsettings.json", optional: false)
-            .AddEnvironmentVariables()
-            .Build();
+            .AddUserSecrets<ConfigAccessor>()
+            .AddEnvironmentVariables();
+        
+        builder.ConfigureAzureKeyVault();
+
+        return _root ??= builder.Build();
```

Comment:
_root is never defined so always null. Should we remove it or lazy initialize it?

Reply:
ye I wasn't too sure about this one, as it's static I assumed it's so it doesnt need renewing every time, I need to understand this a bit more. I'll remove it if it's not doing anything